lib/img/Screenshot.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+export const Screenshot = ({ title }: { title?: string }) => {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="24"
+      height="24"
+      viewBox="0 0 24 24"
+    >
+      {title !== undefined ? <title>{title}</title> : null}
```

Comment:
Nice touch :)

Reply:
As it looks in other Icon files ;)

Reply:
I think @steabert forgot that I added this to all buttons/icons like 4 months ago. :smile: